helmfile.d/charts/prometheus-servicemonitor/templates/kube-vip-monitor.yaml

```diff
@@ -0,0 +1,36 @@
+{{- if .Values.kubeVIP.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ printf "%s-%s" (include "prometheus-servicemonitor.fullname" .) "kube-vip" | trunc 63 | trimSuffix "-" }}
+  namespace: {{ .Values.kubeVIP.namespace }}
+  labels:
+{{ include "prometheus-servicemonitor.kube-vip.labels" . | indent 4 }}
+spec:
+  type: {{ .Values.kubeVIP.service.type }}
+  ports:
+    - port: {{ .Values.kubeVIP.service.port }}
+      targetPort: {{ .Values.kubeVIP.service.targetPort }}
+      protocol: TCP
+      name: monitoring
+  selector: {{- toYaml .Values.kubeVIP.service.selectorLabels | nindent 4 }}
```

Comment:
This feels like it should live with the kube-vip installation. Is it an upstream chart that doesn't expose a service like this or why do we need to add it ourselves here?